news/2020-10-22-beatmapping-world-championship-2020-concludes.md

```diff
@@ -0,0 +1,67 @@
+
+Once again, congratulations to all of the team for winning the osu! Beatmapping World Championship 2020!
+
+Please take a look at their Grand Finals entry over [here](https://osu.ppy.sh/beatmapsets/1280467#osu/2659837) !
+
+**The underdog Team Argentina makes it second among the sea of big teams!**
+
+Team Argentina did for sure the most surprising route in this championship, they managed to eliminate 3 big countries in a row! 
+
+Their journey started on the ninth place of the Qualifiers leaderboards, immediately matching them with Team China in Round of 16. Surprisingly, the outcome of the match was pretty close for Team Argentina, who won against Team China with a score of 3-2! 
+
+They wanted to show the world that this win against Team China wasn't just a spark of luck, so they performed excellently against the big teams of Team France and Team Brazil, by eliminating them with a 6-1 score, twice in a row! 
+
+Their journey ended against Team United Kingdom in the Grand Finals, by losing to them 4-3, giving United Kingdom their first close score in the competition. 
+
+Please take a look at their Grand Finals entry over [here](https://osu.ppy.sh/beatmapsets/1280480#osu/2659858) !
+
+**Team Brazil takes the bronze medal back home!**
+
+After a sudden retirement in 2019, Team Brazil came back in the race with a brand new line-up! 
+
+Slow and steady winning the race. This strategy led Brazil to many victories.  They started the contest as the twelfth place in the Qualifiers, matching them with a fifth place Team Sweden for the Round of 16, winning to them 3-2.
+
+They valiantly fought for the third place in Finals against Team Russia with a 4-3 score after having a great route among all these strong teams that all fought hard to reach the top of the grid!
+
+Please take a look at their Finals entry over [here](https://osu.ppy.sh/beatmapsets/1280436#osu/2659781) !
+
+Despite being fourth, we still want to congratulate Team Russia for going that far in the tournament and providing the insane amount of work as the three other teams did!
+
+Don't forget to take a peak at their Finals entry [here](https://osu.ppy.sh/beatmapsets/1281199#osu/2661154)!
+
+---
+
+Badges will be delivered by our unique seagull delivery system to these three teams in few days from today! 
+
+We are very thankful to **all the contestants**, **yaspo**, **Ryuusei Aika**, **Electoz**, **Sonnyc**, **Myxo**, **Stack** and **Seto Kousuke** that did an immense work by judging hundreds of entries while still respecting all the short deadlines. Nothing would have happen without them! Please give a round of applause to **Milan-** that managed to solely handle the website management during the whole period!
```

Comment:
might want to provide links to their profiles?

Reply:
[I've already suggested that](https://github.com/ppy/osu-wiki/pull/4357#discussion_r508645448), and seemed like @Pachiru has just applied them. Perhaps you're looking at an outdated version of the file.

Reply:
probably, i had an option to refresh the page while reviewing the news post 🤷